x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -229,6 +232,29 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
         broker_timeout: { type: 'integer' },
         broker_ack_reliability: { type: 'text' },
         broker_buffer_size: { type: 'integer' },
+        required_acks: { type: 'integer' },
```

Comment:
We should probably add `index: false` to all of those kafka fields as it seems to me that we will not search on that